modules/Header/ui/CategoriesNav/FeaturedCategory.tsx

```diff
@@ -0,0 +1,59 @@
+import type { Category, TranslatedCategory } from '@prezly/sdk';
+import classNames from 'classnames';
+
+import { type CardSize, CategoryImage } from '@/components/CategoryImage';
+import { Link } from '@/components/Link';
+
+import styles from './FeaturedCategory.module.scss';
+
+export function FeaturedCategory({
+    category,
+    translatedCategory,
+    size,
+    className,
+}: FeaturedCategory.Props) {
+    return (
+        <div className={classNames(className, styles.container)}>
+            <Link
+                className={styles.image}
+                href={{
+                    routeName: 'category',
+                    params: {
+                        slug: translatedCategory.slug,
+                        localeCode: translatedCategory.locale,
+                    },
+                }}
+            >
+                <CategoryImage
```

Comment:
I forgot to report this but the images can become uneven:

![Screenshot 2024-03-20 at 13 14 43](https://github.com/prezly/theme-nextjs-bea/assets/4209081/8ac9f8f9-f675-4a84-978a-eb0c10893db6)

Reply:
570bd301